src/plugins/intel_gpu/tests/test_cases/fully_connected_gpu_test.cpp

```diff
@@ -1591,10 +1591,10 @@ INSTANTIATE_TEST_SUITE_P(
     fully_connected_i8_u8_test,
     testing::Combine(
         testing::Values(1, 2),
-        testing::Values(3, 64),
+        testing::Values(16, 64),
         testing::Values(1),
         testing::Values(1),
-        testing::Values(3, 32),
+        testing::Values(16, 32),
```

Comment:
could you describe more in the commit message why it is changed?

Reply:
As discussed, I added some  description about this change.

Reply:
NIT: Maybe you change the shape, because some specific shape has the issue in oneDNN side, if true, could you add a comment in failed case? It should be help to meet unexpected error in the later.

Reply:
At least, I was fine on benchdnn execution. A relevant comment is added to commit message.